src/firecracker/examples/uffd/uffd_utils.rs

```diff
@@ -162,6 +162,110 @@ impl UffdPfHandler {
     }
 }
 
+#[derive(Debug)]
+pub struct Runtime {
+    stream: UnixStream,
+    backing_file: File,
+    backing_memory: *mut u8,
+    backing_memory_size: usize,
+    uffds: HashMap<i32, UffdPfHandler>,
+}
+
+impl Runtime {
+    pub fn new(stream: UnixStream, backing_file: File) -> Self {
```

Comment:
NIt: It would be preferable to propagate the error but given this is an example its okay either way.
```suggestion
    pub fn new(stream: UnixStream, backing_file: File) -> Result<Self, NewRuntimeError> {
```